src/server.ts

```diff
@@ -128,7 +196,11 @@ export class SubscriptionServer {
 
   private unsubscribe(connectionContext: ConnectionContext, reqId: string) {
```

Comment:
you can keep this as unsubscribe since you will be unsubscribing from observable anyway.